common/djangoapps/track/request_id_utils.py

```diff
@@ -0,0 +1,44 @@
+"""
+Helper functions to access and update the id and type
+used in event tracking.
+"""
+from uuid import uuid4, UUID
+from request_cache import get_cache
+
+
+def get_user_action_id():
+    """
+    Retrieves the current user action id from the request
+    cache.
+    """
+    return get_cache('grade_tracking').get('user_action_id', None)
```

Comment:
While the utilities are in the `common` folder (which is good), the cache name used here is `grades` specific.  Need to correct this.